Fix intermittent docker build errors

#### Context

Sometimes on various branches including `main` , we are failing to build the docker images in the node-build Github Action, particularly the `arm64` builds:

```

#11 [base 6/6] RUN --mount=type=cache,id=pnpm,target=/pnpm/store     pnpm fetch     | grep -v "cross-device link not permitted\|Falling back to copying packages from store"
#11 6.752 Importing packages to virtual store
#11 23.01 qemu: uncaught target signal 4 (Illegal instruction) - core dumped
#11 24.93 Already up to date
#11 24.93 Progress: resolved 1, reused 0, downloaded 0, added 0
#11 24.93 Progress: resolved 2561, reused 0, downloaded 0, added 0
#11 24.93 Progress: resolved 2561, reused 0, downloaded 1, added 0
#11 24.93 Progress: resolved 2561, reused 0, downloaded 16, added 10
#11 24.93 Illegal instruction (core dumped)
#11 DONE 25.6s

#12 [prod-deps 1/4] COPY package.json pnpm-workspace.yaml .npmrc ./
#12 DONE 0.1s

#13 [builder 1/4] COPY package.json pnpm-workspace.yaml .npmrc tsconfig.json tsconfig.build.json ./
#13 DONE 0.1s

#14 [prod-deps 2/4] COPY packages/card-service/package.json ./packages/card-service/package.json
#14 DONE 0.0s

#15 [builder 2/4] COPY packages/card-service ./packages/card-service
#15 DONE 0.0s

#16 [builder 3/4] RUN --mount=type=cache,id=pnpm,target=/pnpm/store     pnpm install     --recursive     --offline     --frozen-lockfile
#16 1.533 ! Corepack is about to download https://registry.npmjs.org/pnpm/-/pnpm-8.15.9.tgz
#16 ...

#17 [prod-deps 3/4] RUN pnpm clean
#17 1.722 ! Corepack is about to download https://registry.npmjs.org/pnpm/-/pnpm-8.15.9.tgz
#17 ...

#16 [builder 3/4] RUN --mount=type=cache,id=pnpm,target=/pnpm/store     pnpm install     --recursive     --offline     --frozen-lockfile
#16 12.94 Scope: all 2 workspace projects
#16 14.66 Lockfile is up to date, resolution step is skipped
#16 ...

#17 [prod-deps 3/4] RUN pnpm clean
#17 12.98 
#17 12.98 > @interledger/rafiki@ clean /home/rafiki
#17 12.98 > find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
#17 12.98 
#17 DONE 15.5s

#18 [prod-deps 4/4] RUN --mount=type=cache,id=pnpm,target=/pnpm/store     pnpm install     --recursive     --prefer-offline     --frozen-lockfile     --prod     | grep -v "cross-device link not permitted\|Falling back to copying packages from store"
#18 ...

#16 [builder 3/4] RUN --mount=type=cache,id=pnpm,target=/pnpm/store     pnpm install     --recursive     --offline     --frozen-lockfile
#16 17.95 Progress: resolved 1, reused 0, downloaded 0, added 0
#16 20.34 .                                        | +735 ++++++++++++++++++++++++++++++++
#16 20.48 Progress: resolved 735, reused 0, downloaded 0, added 0
#16 20.64  ERR_PNPM_NO_OFFLINE_TARBALL  A package is missing from the store but cannot download it in offline mode. The missing package may be downloaded from https://registry.npmjs.org/@adonisjs/fold/-/fold-8.2.0.tgz.
#16 ERROR: process "/dev/.buildkit_qemu_emulator /bin/sh -c pnpm install     --recursive     --offline     --frozen-lockfile" did not complete successfully: exit code: 1

#18 [prod-deps 4/4] RUN --mount=type=cache,id=pnpm,target=/pnpm/store     pnpm install     --recursive     --prefer-offline     --frozen-lockfile     --prod     | grep -v "cross-device link not permitted\|Falling back to copying packages from store"
```

Some examples of this: 

* [main branch](https://github.com/interledger/rafiki/actions/runs/18869711252/job/53845048265)
* [pos-card-services branch](https://github.com/interledger/rafiki/actions/runs/18879873662/job/53879708144?pr=3714)

<br>We should determine the root cause and add a fix for the build or the Dockerfiles to prevent these errors from happening.

--- a/.github/workflows/node-build.yml
+++ b/.github/workflows/node-build.yml
@@ -457,10 +457,6 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
-        with:
-          platforms: ${{ matrix.platform.arch }}
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
         with:

